backend/populate_reservations.py

```diff
@@ -200,6 +200,7 @@ def create_guests(init_file="", init_rooms=""):
     characters = string.ascii_letters + string.digits + string.punctuation
 
     for guest_new in dr:
+        print(f'new guest: {guest_new}')
```

Comment:
Eventually we will want to have some stable logging

Reply:
Ja all the `/views/` have python logging setup. 
If we decide to keep using this script adding logging would be good but if we move most the functionality to a /admin page prolly best to wait(?)